src/DetailsView/components/cards-view.tsx

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as React from 'react';
+
+import { NamedSFC } from '../../common/react/named-sfc';
+
+export interface CardsViewProps {}
+
+export const CardsView = NamedSFC<CardsViewProps>('CardsView', props => {
+    return (
+        <>
+            <h2>Cards view</h2>
+            <p>Hello, World!</p>
+</>
```

Comment:
nit: spacing here seems to be off. Do you have the suggested extensions installed? Prettier should format on file save.